Fl 4

DaveKnifeman opened the pull request.

Davidmesman committed:
Added the block animation for the expertise section

Davidmesman committed:
Added the block animation for the expertise section

Commit:
Merge branch 'develop' into fl-4

Commit:
Merge branch 'develop' into fl-4

SjaqielM commented on index.html:
```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <title>Fairtrade</title>
     <link rel="stylesheet" href="css/main.css">
+    <style id="animationStyles"></style>
```
waarom een style tag?

SjaqielM reviewed (changes requested):
javascript animations don't work

Commit:
changed the way how blocks fade in and fade out

Commit:
changed the way how blocks fade in and fade out

Commit:
Merge branch 'develop' into fl-4

Commit:
Update index.html

Commit:
Update index.html

SjaqielM committed:
style footer

fixed style footer

Commit:
merge index.html and main.scss

Commit:
merge index.html and main.scss

SjaqielM committed:
create expertise section

create section, and make it like the design
back-end javascript will be added later

SjaqielM committed:
Merge remote-tracking branch 'origin/fl-4' into fl-4

# Conflicts:
#	main.scss

SjaqielM committed:
2.0 fixed

the blocks are now center.
the task is done
you done now

Commit:
Merge branch 'develop' of github.com:1022792/projectfair into fl-4

Commit:
Merge branch 'fl-4' of github.com:1022792/projectfair into fl-4

Commit:
add animation on scroll on the blocks
